### PR TITLE
[14.0][FIX] agreement_legal: fix action open agreement

### PR DESCRIPTION
**Pull request by ps-tubtim**
This PR fixes **Access Error** when click smart button **Agreements** on Contact Form View.

Selection_121

### CODE REVIEW

File: agreement_legal/models/res_partner.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def _compute_agreements_count(self):
     def action_open_agreement(self):
         self.ensure_one()
         action = self.env.ref("agreement.agreement_action")
-        result = action.read()[0]
+        result = action.sudo().read()[0]
```
Comment (pedrobaeza) [inline agreement_legal/models/res_partner.py]: A shorter alternative:

```suggestion
        action = self.env["ir.actions.actions"]._for_xml_id("agreement.agreement_action")
```

Comment (ps-tubtim) [reply agreement_legal/models/res_partner.py]: Thank you. I already fixed it.

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
@ps-tubtim Could you check pre-commit ?

**Pull request merged**